src/protocol/spi/nrf24l01.c

```diff
@@ -88,6 +88,7 @@ u8 NRF24L01_WritePayload(u8 *data, u8 length)
     {
         PROTOSPI_xfer(data[i]);
     }
+    usleep(1);  // makes sure SPI transfer has completed
```

Comment:
there is no flag to check if the transfer is finished? sleep 1 us is not safe enough.

Reply:
I don't know if there's a flag, I'm sure all we need is a waiting a couple of cycles, a few nops would probably be enough, I've seen this happen only once.

Reply:
Well, it's already supposed to wait already 
/* Wait for transfer finished. */
while (!(SPI_SR(spi) & SPI_SR_RXNE));

Reply:
Well, it's already supposed to wait
https://github.com/DeviationTX/libopencm3/blob/7e1d3daa113e1ae21f308759e50be2f89361d90f/lib/stm32/common/spi_common_all.c#L236